build-support/bin/release.sh

```diff
@@ -11,17 +11,14 @@ source "${ROOT}/build-support/common.sh"
 
 # Note we allow the user to predefine this value so that they may point to a specific interpreter.
 export PY="${PY:-python3.6}"
-interpreter_constraint="CPython==3.6.*"
 if ! command -v "${PY}" >/dev/null; then
   die "Python interpreter ${PY} not discoverable on your PATH."
 fi
 py_major_minor=$(${PY} -c 'import sys; print(".".join(map(str, sys.version_info[0:2])))')
-if [[ "${py_major_minor}" != "3.6" ]]; then
-  die "Invalid interpreter. The release script requires Python 3.6 (you are using ${py_major_minor})."
+if [[ "${py_major_minor}" != "3.6" || "${py_major_minor}" != "3.7" ]]; then
+  die "Invalid interpreter. The release script requires Python 3.6 or 3.7 (you are using ${py_major_minor})."
```

Comment:
I think it's fine to run this script with Py37? Iirc, We originally didn't allow it because the wheel wouldn't work for Py37.